synedrion/Cargo.toml

```diff
@@ -10,34 +10,38 @@ readme = "README.md"
 categories = ["cryptography", "no-std"]
 
 [dependencies]
-signature = { version = "2", default-features = false, features = ["alloc"] }
-k256 = { version = "0.13.2", default-features = false, features = ["ecdsa", "arithmetic"] }
-rand_core = { version = "0.6.4", default-features = false }
-sha2 = { version = "0.10", default-features = false }
-sha3 = { version = "0.10", default-features = false }
-digest = { version = "0.10", default-features = false, features = ["alloc"]}
+signature = { version = "2.3.0-pre.4", default-features = false, features = ["alloc"] }
+k256 = {git = "https://github.com/RustCrypto/elliptic-curves", default-features = false, features = ["ecdsa", "arithmetic"]}
```

Comment:
I would really like to merge with this pointing to a published version, but Tony seems to be waiting for something. I would rather not keep this PR open any longer, but I think we should at least point to a specific commit here.

Reply:
I'll remind him again.

Reply:
🙏